transform/mattermost-analytics/models/intermediate/sales/hightouch/_int_hightouch__models.yml

```diff
@@ -0,0 +1,42 @@
+version: 2
+
+models:
+  - name: int_onprem_subscriptions
+    description: |
+      Reconstructed `onprem_subscriptions` table using Stripe events.
```

Comment:
is it really "onprem_subscriptions" the right name since its main focus is opportunities? Or maybe splitting the two - create a separate `onprem_subscriptions` staging or intermediate model and source it from an `int_onprem_opportunities` model? cc: @ifoukarakis

Reply:
I originally thought that this table is a mapping between subscriptions and opportunities. However, if I get it right, it's a denormalized join of all subscriptions, invoices and invoice line items. 

Perhaps the name could be `int_denormalized_invoice_items`.

Reply:
Although it is denormalized, it is not all invoice items. this only contains Self-Hosted data. Maybe we could use, `int_onprem_invoice_items`

Reply:
We'll prob. need to converge on a naming convention for these models - are we using the source entity names (e.g. invoices) or the destination entity names (e.g. opportunities, subscriptions, etc.) cc: @sri-byte , @ifoukarakis , @munish7771